shared/src/main/kotlin/com/egm/stellio/shared/model/NgsiLdEntity.kt

```diff
@@ -358,7 +358,7 @@ fun parseToNgsiLdAttributes(attributes: Map<String, Any>): List<NgsiLdAttribute>
     }
 
 fun String.isNgsiLdSupportedName() =
-    this.all { char -> char.isLetterOrDigit() || listOf(':', '_').contains(char) }
+    this.all { char -> char.isDefined() && !listOf('<', '"', '\'', '=', ';', '(', ')', '>', '?', '*').contains(char) }
```

Comment:
you need to rebase your branch to remove this hack commit (done temporarily for https://github.com/FIWARE/load-tests/issues/90)

Reply:
I don't know if the result is the same but I did a revert of this commit